addons/sale_expense/tests/test_reinvoice.py

```diff
@@ -9,109 +9,227 @@
 @tagged('-at_install', 'post_install')
 class TestReInvoice(TestExpenseCommon, TestSaleCommon):
 
-    def test_expenses_reinvoice(self):
-        (self.company_data['product_order_sales_price'] + self.company_data['product_delivery_sales_price']).write({
-            'can_be_expensed': True,
+    @classmethod
+    def setUpClass(cls, chart_template_ref=None):
+        super().setUpClass(chart_template_ref=chart_template_ref)
+        cls.company_data.update({
```

Comment:
why does it need to be defined in company_data? 
`cls.service_order_sales_price = ...`